lib/orderbook/errors.ts

```diff
@@ -65,6 +66,10 @@ const errors = {
     message: `${currency} outbound balance of ${availableAmount} is not sufficient for order amount of ${amount}`,
     code: errorCodes.INSUFFICIENT_OUTBOUND_BALANCE,
   }),
+  EXCEEDING_LIMIT: (currency: string, amount: number, limit: number) => ({
+    message: `outbound amount: ${amount} is exceeding limit of: ${limit} set for ${currency}`,
```

Comment:
The extra `:` in here make this message kind of hard to read, I think we can just drop both of them.